common-compose.yml

```diff
@@ -0,0 +1,36 @@
+# MySQL designed for container-native deployment on Joyent's Triton platform.
+
+mysql:
+    image: 0x74696d/mysql:latest
+    mem_limit: 4g
+    restart: always
+    env_file: .env
```

Comment:
Honestly I just took this from the example stanza in https://docs.docker.com/compose/compose-file/#env-file. I do kinda like having files with credentials as dotfiles because then you're less likely to copy them around unintentionally. But I'm not married to it, as the `.gitignore` *should* prevent you from committing them at least.